lutron-status.js

```diff
@@ -17,20 +18,20 @@ module.exports = function (RED) {
           /*
           for dimmer action is always 1
           for pico action.param
-            FullOn => a=2, p=4
-            up     => a=5, p=4
-            down   => a=6  p=4
-            off    => a=4, p=4
+            FullOn => a=2, p=3
+            up     => a=5, p=3
+            down   => a=6  p=3
+            off    => a=4, p=3
           for on off switch
             action =1 p=0 or 100
           */
           if (action == '1') {
             // either dimmre of switch
             node.send({
-              payload: value
+              payload: node.sendObj ? d : value
             });
-          } else if (value === 4) {
-            var m = '';
+          } else if (value === 3 || node.sendObj) {
```

Comment:
This should likely test `|| (value === 4 && node.sendObj)`. I know the only documented actions for pico remotes are 3 and 4 but that's no reason to make assumptions.